src/main.py

```diff
@@ -56,3 +60,7 @@
         # sys.exit(0)
     except ResizeScreenError as e:
         last_scene = e.scene
+    except StopApplication:
+        quit(0)
+    except KeyboardInterrupt:
+        quit(0)
```

Comment:
should close the screen before quiting

Reply:
Done 310f53f86db25b67229ea2ca6bf4086bad3da31e